server/models/01-schema.sql

```diff
@@ -2569,6 +2570,20 @@ CREATE TABLE `staffing_grade_indice` (
   CONSTRAINT `staffing_grade_indice__grade` FOREIGN KEY (`grade_uuid`) REFERENCES `grade` (`uuid`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
+DROP TABLE IF EXISTS `rubric_grade_indice`;
+CREATE TABLE `rubric_grade_indice` (
+  `uuid` BINARY(16) NOT NULL,
+  `value`  DECIMAL(19,4) NOT NULL,
+  `grade_uuid` BINARY(16) NOT NULL,
+  `rubric_id` INT(10) UNSIGNED NOT NULL,
+  PRIMARY KEY (`uuid`),
+  KEY `rubric_id` (`rubric_id`),
+  KEY `grade_uuid` (`grade_uuid`),
+  CONSTRAINT `rubric_grade_indice__grade` FOREIGN KEY (`grade_uuid`) REFERENCES `grade` (`uuid`),
+  CONSTRAINT `rubric_grade_indice__rubric_payroll` FOREIGN KEY (`rubric_id`) REFERENCES `rubric_payroll` (`id`),
+  UNIQUE KEY `rubric_grade_indice_1` (`grade_uuid`, `rubric_id`)
```

Comment:
If I understand your explanation correctly, the relationship between grades and rubrics is many-to-one.  That is, multiple grades can have the same rubric, but each grade can only have a single rubric.

If that is true, do we really need a whole other table for this?  Why can't we make it part of the `grade` table?

Reply:
I’m afraid not. Regardless of the case, it will always be necessary to have another table because for the same grade, there will be specific values for multiple items. For instance, grade A1 will have a value for overtime, another for call bonus, and yet another for holiday pay. This results in a 1.N and 1.N cardinality, hence the need to create a new table. The restriction I’ve applied with UNIQUE ensures that there are no duplicate grade-item pairs, such as 
````{grade: 'A1', item: 'night', value: 0.2} and {grade: 'A1', item: 'night', value: 2}.````

Reply:
I understand 👍